source/administration/command-line-tools.rst

```diff
@@ -0,0 +1,661 @@
+   name would be ``core`` and channel name would be ``town-square``
+
+.. tip :: If you automate user creation through the CLI tool with SMTP
+enabled emails will be sent to all new users created. If you run such a
+load script, it is best to disable SMTP or to use test accounts so that
+new account creation emails aren't unintentionally set to people at your
+organization who aren't expecting them.
+
+platform
+~~~~~~~~
+Commands for configuring and managing your Mattermost instance and users.
+
+**Options**
+
+::
+
+      -c, --config string   Configuration file to use. (default "config.json")
+
+**Child Commands**
+
+-  `platform channel`_ - Management of channels
+-  `platform help`_ - Generate full documentation for the CLI
+-  `platform import`_ - Import data
+-  `platform ldap`_ - LDAP related utilities
+-  `platform license`_ - Licensing commands
+-  `platform reset`_ - Reset the database to initial state
+-  `platform roles`_ - Management of user roles
+-  `platform server`_ - Run the Mattermost server
+-  `platform team`_ - Management of teams
+-  `platform user`_ - Management of users
+-  `platform version`_ - Display version information
+
+platform channel
+~~~~~~~~~~~~~~~~
+
+Commands for channel management.
+
+**Child Commands**
+
+-  `platform channel add`_ - Add users to a channel
+-  `platform channel create`_ - Create a channel
+-  `platform channel delete`_ - Delete a channel
+-  `platform channel list`_ - List all channels on specified teams
+-  `platform channel remove`_ - Remove users from a channel
+-  `platform channel restore`_ - Restore a channels
+
+platform channel add
+^^^^^^^^^^^^^^^^^^^^
+
+Add users to a channel.
+
+::
+
+    platform channel add [channel] [users]
+
+**Examples**
+
+::
+
+      channel add mychannel user@example.com username
+
+platform channel create
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a channel.
+
+::
+
+    platform channel create
+
+**Examples**
+
+::
+
+      channel create --team myteam --name mynewchannel --display_name "My New Channel"
+      channel create --team myteam --name mynewprivatechannel --display_name "My New Private Channel" --private
+
+**Options**
+
+::
+
+          --display_name string   Channel Display Name
+          --header string         Channel header
+          --name string           Channel Name
+          --private               Create a private channel.
+          --purpose string        Channel purpose
+          --team string           Team name or ID
+
+platform channel delete
+^^^^^^^^^^^^^^^^^^^^^^^
+Permanently deletes a channel along with all related information,
+including posts from the database. Channels can be specified by
+[team]:[channel] using the team and channel names or IDs.
+
+::
+
+    platform channel delete [channels]
+
+**Examples**
+
+::
+
+      channel delete myteam:mychannel
+
+platform channel list
+^^^^^^^^^^^^^^^^^^^^^
+
+List all channels on a specified team. Archived channels are appended with `` (archived)``.
+
```

Comment:
This doesn't get displayed correctly:
![image](https://cloud.githubusercontent.com/assets/20325173/21911921/d2b81490-d8d7-11e6-9e9c-ec231075a80e.png)

To fix, remove the space between the back ticks and the parens: ` ``(archived)`` `